modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -2948,6 +2948,113 @@ definition {
 			locator1 = "ObjectAdmin#VIEW_ACTIONS_LABEL");
 	}
 
+	@description = "LPS-165849 - Verify terms from a related custom object (child) are replaced correctly in an email notification"
+	@priority = 5
+	test CanSendEmailNotificationWithTermsComingFromCustomObjectRelationship {
+		property custom.properties = "feature.flag.LPS-165849=true";
+		property custom.properties = "feature.flag.LPS-171625=true";
+		property portal.acceptance = "true";
+		property test.smtp.server.enabled = "true";
+
+		task ("Given custom object A with a 1:M relationship to custom object B") {
+			for (var letters : list "A,B") {
+				ObjectAdmin.addObjectViaAPI(
+					labelName = "Custom Object ${letters} 182605",
+					objectName = "CustomObject${letters}182605",
+					pluralLabelName = "Custom Objects ${letters} 182605");
+
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Field Text ${letters}",
+					fieldName = "customField",
+					fieldType = "String",
+					isRequired = "false",
+					objectName = "CustomObject${letters}182605");
+
+				ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject${letters}182605");
+			}
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObjectA182605",
+				objectName_2 = "CustomObjectB182605",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+		}
+
+		task ("And given a email notification template using terms from both object definitions") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				emailBody = "[%CUSTOMOBJECTB182605_CUSTOMFIELD%]",
```

Comment:
Please use the Object A custom field term here as well. We need to make sure the custom fields terms work too.

Reply:
Only the term from the B custom object entry worked. The one for Object A didn't work. I need to know if this was really expected behavior. @JhossephG